### PR TITLE
 Implement smooth panscrolling

### PR DESCRIPTION
**Pull request by whot**
This is the remaining patchset from #222 squashed together, after #297 and #296. No changes other than a whitespace fix and some comment typo.

@Greenscreener, can you give this one a test please to make sure nothing broke? Thanks

### CODE REVIEW

File: src/x11/xf86Wacom.c [hunk 1]
```diff
@@ -397,11 +397,9 @@ valuatorNumber(enum WacomAxisType which)
 }
```
Comment (Greenscreener) [inline src/x11/xf86Wacom.c:397]: While resolving conflicts, you forgot to add the scroll axes to the switch in `valuatorNumber()`. See 07581618a0d26a8135cde4fd930c3eb432d62461

Comment (whot) [reply src/x11/xf86Wacom.c:399]: squashed in, thanks!

File: test/test_wacom.py [hunk 2]
```diff
@@ -327,4 +327,59 @@ def test_axis_updates_wheel(mainloop, opts, stylus_type):
...
+    monitor.write_events(prox_out)
+
+    mainloop.run()
+    have_we_scrolled = False
+    for event in monitor.events:
+        if event.axes.scroll_x != 0:
+            assert event.axes.scroll_x == -1223320
+            have_we_scrolled = True
+    assert have_we_scrolled
+
```
[the remark this answers is not in the document]
Comment (whot) [reply test/test_wacom.py:384]: squashed in, thanks!

**Comment by whot** [comment]
Thanks for testing, let's get this merged, we can fix any remaining issues with follow-up patches.

**Comment by Greenscreener** [comment]
Awesome! Thank you very much for your cooperation and help, this is my first proper contribution to open source and I'm glad to have it finally done!

**Pull request merged**